incremental-font-transfer/src/ift_extend.rs

```diff
@@ -0,0 +1,83 @@
+//! IFT Extension
+//!
+//! This command line tool executes the IFT extension algorithm (<https://w3c.github.io/IFT/Overview.html#extend-font-subset>) on an IFT font.
+
+use std::collections::HashMap;
+
+use clap::Parser;
+use incremental_font_transfer::{
+    patch_group::{PatchGroup, UriStatus},
+    patchmap::SubsetDefinition,
+};
+use read_fonts::collections::IntSet;
+use skrifa::FontRef;
+
+#[derive(Parser, Debug)]
+#[command(version, about = "Run the IFT extension algorithm (https://w3c.github.io/IFT/Overview.html#extend-font-subset) on an IFT font.", long_about = None)]
+struct Args {
+    /// The input IFT font file.
+    #[arg(short, long)]
+    font: std::path::PathBuf,
+
+    // The output IFT font file.
+    #[arg(short, long)]
+    output: std::path::PathBuf,
+
+    /// Text to extend the font to cover.
+    #[arg(short, long)]
+    text: String,
+    // TODO(garretrieger): add design space and feature tags arguments.
+}
+
+fn main() {
+    let args = Args::parse();
+
+    let codepoints: IntSet<u32> = args.text.chars().map(|c| c as u32).collect();
+    let subset_definition = SubsetDefinition::codepoints(codepoints);
+
+    let mut font_bytes = std::fs::read(&args.font).unwrap_or_else(|e| {
+        panic!(
+            "Unable to read input font file ({}): {:?}",
+            args.font.to_str().unwrap_or(""),
```

Comment:
`Path` has a method `display()` that returns a `Display` type you can use in this situation.
```suggestion
            args.font.display(),
```